apps/events.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: docs
+key: events
+title: Events
+---
+
+Events are triggered when a specific action is done.
+If you request one of these events, your app will be informed about related actions and can react accordingly.
+This can be helpful if your app needs information like the current page the user is on, or the fact that a product was added to the cart.
+
+These events are currently available:
+
+| Event | Type | Description |
+| - | - |  - |
+| pageview | string | A new page is rendered. Gives information on the path of the page. |
+| product | immutable.js object | A product page is rendered. Gives information on the respective product. |
```

Comment:
@BirgitBader Do we wanna say "product page"? Same for "category page" below...

Reply:
Is it a product detail page then?
Maybe we should go with just "product" and "category".

I'm not too sure about the current explanation "rendered". What exactly happens there? A merchant created or updated a product or category?